Homework_6/Task_2.py

```diff
@@ -0,0 +1,8 @@
+my_list = [1, 5, 68, 0]
+# my_list = [1,2,3,4,6,7,8]
+for i in range(1, len(my_list)):
+    if my_list[i] != my_list[i-1] + 1:
+        print("Non-consecutive number in the list: ", my_list[i])
+        break
+else:
+    print("The list contains consecutive numbers!")
```

Comment:
If I create a longer list, the program work in not a good way
![image](https://github.com/senya2006/Semen-Homework/assets/51053483/87b0ef52-3a81-47b5-887e-1367dde0b121)